pages/18-01.md

```diff
@@ -382,6 +387,49 @@ For all second-level domains and all mail-sending hosts generally, make a plan t
 
 See the [Resources](/resources/#implement) page for implementation case studies.
 
+
+#### How should DMARC be deployed?
+DMARC was developed to enable cautious deployment. While the standard has [three policy states](#alignment-and-conformance) (none, quarantine, and reject), it also has a [`pct`](https://tools.ietf.org/html/rfc7489#section-6.3), or percent, option that a domain owner can use to tell recipients what volume of messages should have the policy applied. When `pct` is left unspecified, the default value of 100% is used.
+
+Rolling DMARC out incrementally by increasing the policy state and percent value over time, while carefully monitoring daily DMARC aggregate reports, is a good strategy to get to p=reject.
+
+One [Google support document](https://support.google.com/a/answer/2466563?hl=en) offers a nominal deployment:
+
+1. Monitor all (p=none)
+2. Quarantine 1%
+3. Quarantine 5%
+4. Quarantine 10%
+5. Quarantine 25%
+6. Quarantine 50%
+7. Quarantine all
+8. Reject 1%
+9. Reject 5%
+10. Reject 10%
+11. Reject 25%
+12. Reject 50%
+13. Reject all
+
+By the [DMARC deadline](#checklist), p=reject must be at `pct=100` either explicitly or implicitly.
+
+
+#### Why is `p=reject` required?
+A primary goal of the Directive is to make it hard to successfully spoof government services.  A DMARC policy of `p=none` allows you to gather insight into those systems that are sending mail using your domain – including imposters. But `p=none` doesn’t actually delay or stop unauthenticated mail from being delivered. It should be viewed as a temporary state on the path to higher levels of enforcement.
+
+`p=quarantine` is where a domain’s policy starts to be enforced. This means that, for most users, unauthenticated mail from a domain with `p=quarantine` would be placed in a junk email folder. Since the message is delivered, though, a user could still find it and be harmed by it.
+
+Only a policy of `p=reject` blocks delivery of unauthenticated message completely.
+
+
+#### Does using `p=reject` mean that leads or anonymous sources may not be able to send us emails?
+Having legitimate email inadvertently filtered is a common concern for any organization employing traditional spam-filtering technologies. DMARC is fundamentally different from spam-filtering in how it protects from malicious email, however.
```

Comment:
I know this is merged, but as an armchair suggestion - you might want to start off the answer with a clean "**No.**" before going into the explanation.

Reply:
Done.